### DISCUSSION TITLE
Understanding spacing

### DISCUSSION THREAD

**Discussion by emi-dm** [Project-MONAI/MONAI#7470] [Q&A]
Hi, does anyone know where I could find resources to understand volume spacing and be able to standardize it properly with Monai?

Thanks in advanced!

**Comment by KumoLiu** [comment]
Hi @emi-dm, from this [discussion](https://github.com/Project-MONAI/MONAI/discussions/7473?converting=1), I think the problem should not be related with Spacing.

Feel free to ask for further help, thanks.
